src/Routing/Route/Route.php

```diff
@@ -131,6 +131,12 @@ public function __construct(string $template, array $defaults = [], array $optio
         $this->setExtensions((array)$this->options['_ext']);
         $this->setMiddleware((array)$this->options['_middleware']);
         unset($this->options['_middleware']);
+
+        if (isset($this->defaults['_method'])){
+            $this->defaults['_method'] = is_array($this->defaults['_method'])
```

Comment:
Should we just force this to be an array?

Reply:
We _could_ but it is a change in behavior and could break application code as `defaults` is a public property. I briefly had changed it to an array when working on this but quickly reverted as many tests failed.